### PR TITLE
SISRP-10788 Fix conditional decision to show profile link

### PR DESCRIPTION
**Pull request by ctweney**
https://jira.berkeley.edu/browse/SISRP-10788

This JIRA blocks all performance testing so I plan to self-merge once T is happy.

### CODE REVIEW

**Comment by ctweney** [comment]
To clarify, this makes gear_popover.html use the same conditional used by profile.html. That means the backend makes the decision of show/not-show profile, as it should do.

File: src/assets/templates/widgets/gear_popover.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <h4 data-ng-bind="api.user.profile.fullName"></h4>
     </div>
     <ul class="cc-popover-items">
-      <li class="cc-popover-item" data-ng-if="api.user.profile.showSisProfileUI && api.user.profile.roles.student">
+      <li class="cc-popover-item" data-ng-if="api.user.profile.showSisProfileUI">
```
Comment (christianvuerings) [inline src/assets/templates/widgets/gear_popover.html]: @ctweney this change lgtm but does this mean we aren't setting the `roles.student` correctly?

Comment (ctweney) [inline src/assets/templates/widgets/gear_popover.html]: Yes. That will be a followup JIRA.

Comment (ctweney) [inline src/assets/templates/widgets/gear_popover.html]: The followup is https://jira.berkeley.edu/browse/SISRP-10789

**Pull request merged**